netmanager/src/views/pages/UserList/UserList.js

```
@@ -31,17 +38,62 @@ const UserList = (props) => {
+    try {
+      const res = await getNetworkUsersListApi(activeNetwork._id, {
+        page: page + 1,
+        limit
+      });
+      setUsers(res.assigned_users);
+      setTotalCount(res.total || 0);
+    } catch (error) {
+      let errorMessage = 'An error occurred';
+      if (error.response && error.response.status >= 500) {
+        errorMessage = 'An error occurred. Please try again later';
+      } else if (error.response && error.response.data && error.response.data.message) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)